lib/mayaUsd/resources/ae/usdschemabase/__init__.py

```diff
@@ -1,5 +1,5 @@
 from .ae_template import AETemplate
-from .custom_enum_control import customEnumControlCreator
+from .enumCustomControl import EnumCustomControl
```

Comment:
This could potentially impact external scripts that use the old naming.

Reply:
I doubt user scripts were referring to internal classes used to create the AE UI. Assuming otherwise would prevent us from ever modifying code.